ksqldb-functional-tests/src/test/resources/query-validation-tests/json_sr.json

```diff
@@ -865,6 +865,20 @@
         "type": "io.confluent.ksql.util.KsqlStatementException",
         "message": "JSON only supports MAP types with STRING keys"
       }
+    },
+    {
+      "name": "struct must be converted to String",
+      "statements": [
+        "CREATE STREAM TEST (FOO STRUCT <F0 INT>) WITH (kafka_topic='test_topic', value_format='JSON_SR', partitions=1);",
+        "CREATE STREAM INPUT (FOO VARCHAR) WITH (kafka_topic='test_topic', value_format='JSON_SR');",
+        "CREATE STREAM OUTPUT as select * from INPUT;"
+      ],
+      "inputs": [
+        {"topic": "test_topic", "value": {"FOO": {"F0": 1}}}
+      ],
+      "outputs": [
+        {"topic": "OUTPUT", "value": {"FOO": "Struct{F0=1}"}}
```

Comment:
Could you check a plain JSON format to see how the Struct looks like? I think the plain JSON is only `{F0=1}` (withouth the Struct word).

Reply:
> Could you check a plain JSON format to see how the Struct looks like? I think the plain JSON is only `{F0=1}` (withouth the Struct word).

The plain json does not have the word `Struct` (As you told). I believe that the word `Struct` comes from the `toString()` method implemented by the `Struct` class: `org.apache.kafka.connect.data.Struct.toString()`

Reply:
but `protobuf` and `avro` show the word Struct as well.